install.ps1

```
@@ -0,0 +1,110 @@
+Write-Info "  to:   $BinaryPath"
+Write-Host ""
+
+if (-not (Test-Path $InstallDir)) {
+    New-Item -ItemType Directory -Force -Path $InstallDir | Out-Null
+}
+
+try {
+    Invoke-WebRequest -Uri $DownloadUrl -OutFile $BinaryPath -UseBasicParsing
+} catch {
+    if ($_.Exception.Response.StatusCode -eq 404) {
+        Write-Host "error: Release asset not found: $AssetName ($Version)" -ForegroundColor Red
```

PSScriptAnalyzer (1.25.0)

[warning] File 'install.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)